docs/guides/quick_start.mdx

```diff
@@ -90,22 +90,35 @@ Add PlaywrightCrawler gif here.
 </TabItem>
 </Tabs>
 
-By default, Crawlee stores data to `./crawlee_storage` in the current working directory. You can override this behavior by setting the `CRAWLEE_STORAGE_DIR` environment variable. For details, see [Environment variables](../guides/environment-variables), [Request storage](../guides/request-storage) and [Result storage](../guides/result-storage).
+By default, Crawlee stores data to `./crawlee_storage` in the current working directory. We can override this behavior by setting the `CRAWLEE_STORAGE_DIR` environment variable.
 
-For more examples showcasing various features of Crawlee, see the [Examples](../examples) section of the documentation.
+More examples showcasing various features of Crawlee could be found in [Examples](../examples) section of the documentation.
+
+**Related links**
+
+- [Environment variables](../guides/environment-variables)
+- [Request storage](../guides/request-storage)
+- [Result storage](../guides/result-storage)
 
 ## Local usage with Crawlee command-line interface (CLI)
 
-To create a boilerplate of your project, you can use the [Crawlee command-line interface (CLI)](https://...) tool by running:
+To create a boilerplate of our project, we can use the [Crawlee CLI](https://www.npmjs.com/package/@crawlee/cli) tool by running:
+
+```bash
+npx crawlee create my-cheerio-crawler
```

Comment:
Does this actually point to the right tool? I know in the case of apify-cli I had to do `npx apify-cli` instead 😅

Reply:
Yes it does, the metapackage defines a CLI script too.

https://github.com/apify/apify-ts/blob/master/packages/crawlee/package.json#L9